benchmarks/bench-switch.py

```diff
@@ -3,8 +3,8 @@
 import eigenpy
 import numpy as np
 
-import time
-import timeit
+import time  # noqa
+import timeit  # noqa
```

Comment:
Couldn't we remove those imports ?

Reply:
When the benchmark is run with IPython it is being used but the linter cannot figure this out